src/commands/flex/plugins/diff.ts

```diff
@@ -0,0 +1,121 @@
+import { Difference } from 'flex-plugins-api-toolkit/dist/tools/diff';
+
+import { createDescription } from '../../../utils/general';
+import FlexPlugin, { ConfigData, SecureStorage } from '../../../sub-commands/flex-plugin';
+import { TwilioCliError } from '../../../exceptions';
+import { isNullOrUndefined } from '../../../utils/strings';
+
+/**
+ * Configuration sid parser
+ * @param input the input from the CLI
+ */
+export const parser = (input: string) => {
+  if (input === 'active') {
+    return input;
+  }
+
+  if (!input || !input.startsWith('FJ')) {
+    throw new TwilioCliError(`Identifier must of a ConfigurationSid instead got ${input}`);
```

Comment:
`Identifier must be a ConfigurationSid...; instead got...`

but why not:
```
if (!input || !input.startsWith('FJ') || input !== 'active') {
  throw new TwilioCliError(`Identifier must be 'active' or a ConfigurationSid; instead got ${input}`);
}

return input;
```

Reply:
The correct `if` statement would be `if (input !== 'active' && (!input || !input.startsWith('FJ'))` hence why I separated it; (if `input = 'active'` then ` || !input.startsWith('FJ')` is already true and this throws an exception)

Reply:
ahh, but why not just have that if statement you wrote? `Identifier must of a ConfigurationSid instead got` isn't technically true since it could be 'active' too